.travis.yml

```diff
@@ -61,20 +60,18 @@ jobs:
         SDK=python
         SDK_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
         FULLSTACK_TEST_REPO=ProdTesting
-    - stage: 'Test'
-      python: "pypy"
-      before_install:
-        - pip install "cryptography>=1.3.4,<=3.1.1" # installing in before_install doesn't re-install the latest version of the same package in the next stage.
     - stage: 'Test'
       python: "pypy3"
       before_install:
         - pip install "cryptography>=1.3.4,<=3.1.1"
```

Comment:
Do we still need to be less than here with the bumped versions?

Reply:
Yeah, my bad, removed it now. Let's see if pypy3 tests compile